pkg/handler/data_recorder.go

```diff
@@ -14,7 +14,8 @@ var (
 
 // DataRecorder can record and produce the evaluation result
 type DataRecorder interface {
-	AsyncRecord(*models.EvalResult)
+	AsyncRecord(models.EvalResult)
```

Comment:
Seems like there should be some standardization on whether EvalResult is used as a pointer or a value. It seems to be a bit of both right now. Any thoughts on this?

Reply:
When I refactored this interface, I had to make too many nil checks all over the places, and they didn't give much margin benefits especially when the struct is small enough. Instead, I will just assume that we can receive a struct from upstream. Which kind of makes sense: if there's nothing to log, it shouldn't even bother any DataRecorders.

And in the evaluation code path, the ultimate response is auto generated from https://github.com/checkr/flagr/blob/master/swagger_gen/restapi/operations/evaluation/post_evaluation_responses.go#L28, which is a pointer. Therefore the pointer reference is prepared inside eval.go

Reply:
I see, that makes sense. Well that's all I really have from my end. Looks good.